### PR TITLE
Minor refactoring of existing share backup tests

### PR DESCRIPTION
**Pull request by mkoppal-px**
**What this PR does / why we need it**:

- [x] Assigned backup names from an array to a variable instead of using it with index at all places
- [x] Added the right string to verify the Restore disabled error message
- [x] Formatting and alignment issues are fixed in `backup_test.go`

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by mkoppal-px** [comment]
Aetos link for:

- [ShareBackupWithUsersAndGroups](http://aetos.pwx.purestorage.com/resultSet/testSetID/77826)
- [CancelClusterBackupShare](http://aetos.pwx.purestorage.com/resultSet/testSetID/77848)

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Review by skonda-px — APPROVED** [review]
LGTM

**Comment by mkoppal-px** [comment]
Resolved merge conflicts and added the error logging for getting pre-rule UID

```
preRuleUid, err := Inst().Backup.GetRuleUid(orgID, ctx, preRuleNameList[0])
log.FailOnError(err, "Error getting UID for pre-rule - [%s]", preRuleNameList[0])
```

File: tests/backup/backup_test.go [hunk 1]
```diff
@@ -726,8 +727,9 @@ var _ = Describe("{ShareBackupWithUsersAndGroups}", func() {
 			log.FailOnError(err, "Fetching px-central-admin ctx")
 
 			// Share backup with the user
-			err = ShareBackup(backupNames[1], nil, []string{username}, ViewOnlyAccess, ctx)
-			log.FailOnError(err, "Failed to share backup %s", backupNames[1])
+			backupName := backupNames[6]
```
Comment (lsrinivas-pure) [inline tests/backup/backup_test.go:730]: do we have checks if this index is available or not?

**Pull request merged**